docs/source/usage/index.rst

```diff
@@ -15,7 +15,7 @@ ESI accommodates three roles:
 * **Lessee**
    * Can only view nodes that they have leased.
    * Has temporary use of their nodes for the duration of the lease.
-   * Leases out a node from an available offer.
+   * Leases a node from an available offer.
```

Comment:
Since a lessee can't directly create a lease, should this be rephrased to be something more like "Obtains a lease on a node by claiming an available offer"?